components/head-static.html

```diff
@@ -51,8 +51,8 @@
 <script type="module" src="https://unpkg.com/@myuw-web-components/myuw-profile@1.6.3/dist/myuw-profile.min.mjs"></script>
 <script nomodule src="https://unpkg.com/@myuw-web-components/myuw-profile@1.6.3/dist/myuw-profile.min.js"></script>
 
-<script type="module" src="https://unpkg.com/@myuw-web-components/myuw-notifications@1.3.0/dist/myuw-notifications.min.mjs"></script>
-<script nomodule src="https://unpkg.com/@myuw-web-components/myuw-notifications@1.3.0/dist/myuw-notifications.min.js"></script>
+<script type="module" src="https://unpkg.com/@myuw-web-components/myuw-notifications@1.3.1/dist/myuw-notifications.min.mjs"></script>
+<script nomodule src="https://unpkg.com/@myuw-web-components/myuw-notifications@1.3.1/dist/myuw-notifications.min.js"></script>
```

Comment:
@cksteven  I published the newest version to the npm, could you use 1.3.2?

Reply:
Yep! Will do in a bit